src/amo/reducers/viewContext.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_THEME,
+  VIEW_CONTEXT_EXPLORE,
+  VIEW_CONTEXT_HOME,
+  SET_VIEW_CONTEXT,
+} from 'core/constants';
+
+
+export type ViewContextType = {|
+  context: ADDON_TYPE_EXTENSION | ADDON_TYPE_THEME | VIEW_CONTEXT_EXPLORE |
```

Comment:
> I tried this and flow just takes forever to initialise. :-(

I think this is a bug. I've seen it. You can do `yarn flow start` before `yarn flow:dev` to work around it.